### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by reuben**
Rendered: https://github.com/mozilla/DeepSpeech/blob/readme-update/README.md

### CODE REVIEW

**Comment by reuben** [comment]
We've had a bunch of people mixing up training and inference docs, so I moved all the training stuff to the bottom where they're discussed more thoroughly.

File: README.md [hunk 1]
```diff
@@ -54,43 +62,26 @@ See the output of `deepspeech -h` for more information on the use of `deepspeech
...
 
-Running `deepspeech` might require some runtime dependencies to be already installed on your system. Regardless of which bindings you are using, you will need the following:
+Running `deepspeech` might require some runtime dependencies to be already installed on your system:
 
-* libsox2
-* libstdc++6
-* libgomp1
-* libpthread
+* sox - The Python and Node.JS clients use SoX to resample files to 16kHz.
+* libpthread - On Linux, some people have had to install libpthread manually.
```
Comment (lissyx) [inline README.md]: I think we want to keep libstdc++ and libgomp ?

Comment (lissyx) [reply README.md]: And maybe we should add GLIBC versions ?

Comment (reuben) [reply README.md]: Oh, I didn't mean to remove that, will fix.

File: README.md [hunk 2]
```diff
@@ -4,34 +4,40 @@
...
 ```bash
+# Create and activate a virtualenv
+virtualenv -p python3 $HOME/tmp/deepspeech-gpu-venv/
+source $HOME/tmp/deepspeech-gpu-venv/bin/activate
+
+# Install DeepSpeech CUDA enabled package
 pip3 install deepspeech-gpu
+
+# Transcribe an audio file.
 deepspeech --model models/output_graph.pbmm --alphabet models/alphabet.txt --lm models/lm.binary --trie models/trie --audio my_audio_file.wav
```
Comment (kdavis-mozilla) [inline README.md]: Should we also, directly before this bash section, say how to download the example audio which they can use here. Then people will be less likely to make the mistake of feeding it some junk audio and immediately being discouraged.

File: README.md [hunk 3]
```diff
@@ -4,34 +4,40 @@
...
 pip3 install deepspeech
+
+# Transcribe an audio file
 deepspeech --model models/output_graph.pbmm --alphabet models/alphabet.txt --lm models/lm.binary --trie models/trie --audio my_audio_file.wav
 ```
 
-Alternatively, quicker inference can be performed using a supported NVIDIA GPU on Linux. See the [release notes](https://github.com/mozilla/DeepSpeech/releases) to find which GPUs are supported. To run `deepspeech` on a GPU, install the GPU specific package:
+A pre-trained English model is available for use and can be downloaded using [the instructions below](#using-a-pre-trained-model). Currently, only 16-bit, 16 kHz, mono-channel WAVE audio files are supported in the Python client.
+
+Quicker inference can be performed using a supported NVIDIA GPU on Linux. See the [release notes](https://github.com/mozilla/DeepSpeech/releases) to find which GPUs are supported. To run `deepspeech` on a GPU, install the GPU specific package:
```
[the remark this answers is not in the document]
Comment (reuben) [reply README.md]: Good idea.

File: README.md [hunk 4]
```diff
@@ -4,34 +4,40 @@
...
+
+# Install DeepSpeech
 pip3 install deepspeech
+
+# Transcribe an audio file
 deepspeech --model models/output_graph.pbmm --alphabet models/alphabet.txt --lm models/lm.binary --trie models/trie --audio my_audio_file.wav
 ```
 
-Alternatively, quicker inference can be performed using a supported NVIDIA GPU on Linux. See the [release notes](https://github.com/mozilla/DeepSpeech/releases) to find which GPUs are supported. To run `deepspeech` on a GPU, install the GPU specific package:
+A pre-trained English model is available for use and can be downloaded using [the instructions below](#using-a-pre-trained-model). Currently, only 16-bit, 16 kHz, mono-channel WAVE audio files are supported in the Python client.
```
[the remark this answers is not in the document]
Comment (reuben) [reply README.md]: I linked to the parent header on purpose so hopefully people read the intro paragraph as well before blindly downloading.

[the remark this answers is not in the document]
Comment (reuben) [reply README.md]: Will add the intro line to the bash section.

File: README.md [hunk 5]
```diff
@@ -4,34 +4,40 @@
...
-
 ```bash
+# Create and activate a virtualenv
+virtualenv -p python3 $HOME/tmp/deepspeech-venv/
+source $HOME/tmp/deepspeech-venv/bin/activate
+
+# Install DeepSpeech
 pip3 install deepspeech
+
+# Transcribe an audio file
```
[the remark this answers is not in the document]
Comment (reuben) [reply README.md]: It's the sentence that follows this code snippet. Do you think it should come before it?

[the remark this answers is not in the document]
Comment (kdavis-mozilla) [reply README.md]: I think before would help just so they aren't confused as to what these pbmm files are

**Pull request merged**